checkQC/default_config/config.yaml

```diff
@@ -644,6 +747,9 @@ nextseq500_low:
       - name: Q30Handler
         warning: 80
         error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
```

Comment:
This is not a deal breaker since the users can specify this value themselves, but I would prefer if this threshold was 45 (given that cluster PF is 60), to follow the same format as the other default values. If there is a good reason for setting it at 15 I'm not impossible to convince though :) 

See my comment about the ClusterPFHandler thresholds before addressing this.

Reply:
I made back of the envelope calculations, I guess they were wrong ;)

Reply:
I should have adjusted them, but please double check I understood the logic correctly.